src/random.c

```diff
@@ -160,10 +160,10 @@ random_get_uint64 (Random      *random)
 
     return dest;
 }
+
```

Comment:
whitespace

Reply:
Removed the blank line to make the code more unreadable :-(.